backoffice/src/Controllers/MigrationController.cs

```diff
@@ -33,12 +31,54 @@ public IActionResult Index()
         return RedirectToAction("Index", "Home");
     }
     
+    [HttpPost]
+    [RequestTimeout(600000)]
+    public async Task<IActionResult> ProcessTriageFile(IFormFile? file)
+    {
+        if ( file == null || file.Length == 0 )
+        {
+            ModelState.AddModelError("File", "Please select a file");
+            return RedirectToAction("UploadTriageFile");
+        }
+        
+        try
+        {
+            var reader = new StreamReader(file.OpenReadStream());
+            var csv = new CsvReader(reader, CultureInfo.InvariantCulture);
+            var records = csv.GetRecords<TriageRowDto>().ToList();
+
+            var result = await _service.HandleTriageCsvAsync(records);
+            Console.Write("records uploaded");
+            
+            return View("UploadTriageFile", result);
+        }
+        catch ( Exception e )
+        { 
+            HandleError(_logger, "Error updating triage numbers", e);
+
+            return View("UploadTriageFile", new TriageUploadResultDto());
+        }
+
+        
+    }
+
+
+    [HttpGet]
+    public IActionResult UploadTriageFile()
+    {
+        return View(new TriageUploadResultDto());
+    }
     
     
     [HttpPost]
     [AllowAnonymous]
     public async Task<IActionResult> MigrateWreck([FromBody] PowerappsWreckDto request, [FromHeader(Name = "X-API-Key")] string apiKey)
     {
+        if (DisablePowerappsMigrationEndpoints)
+        {

```

Comment:
Basically just turned off all the migration endpoints (flag at top of file if it needs turning back on ever)